src/webgpu/api/validation/queue/copyToTexture/CopyExternalImageToTexture.spec.ts

```diff
@@ -245,7 +241,7 @@ g.test('source_canvas,contexts')
       { texture: dstTexture },
       copySize,
       true, // No validation errors.
-      canCopyFromCanvasContext(contextType) ? '' : 'OperationError'
```

Comment:
These are validation tests. Now they always expect success, should we just delete them in favor of operation tests that actually test the behavior?

Reply:
Removed these test cases